src/_P076_HLW8012.ino

```diff
@@ -492,13 +501,13 @@ boolean Plugin_076(uint8_t function, struct EventStruct *event, String& string)
           # if PLUGIN_076_DEBUG
           addLogMove(LOG_LEVEL_INFO,
                      strformat(F("P076: Read values - V=%.2f - A=%.2f - W=%.2f - VAR=%.2f - VA=%.2f - Pf%%=%.2f - Ws=%.2f"),
-                               HLW[0],
-                               HLW[1],
-                               HLW[2],
-                               HLW[3],
-                               HLW[4],
-                               HLW[5],
-                               HLW[6]));
+                               HLW[P076_INDEX_VOLD],
```

Comment:
Did you compile it?
Should be `P076_INDEX_VOLT`

Reply:
I compiled it but with PLUGIN_076_DEBUG = false. Shame on me.